microcosm_pubsub/producer.py

```diff
@@ -333,8 +333,8 @@ def configure_sns_producer(graph):
         opaque = None
 
     try:
-        register = bool(graph.publish_info_convention)
-    except NotBoundError:
+        register = bool(graph._cache.__getitem__('publish_info_convention'))
```

Comment:
So this tests bound-but-not-created?  
It's pretty subtle.  A comment for future maintainer would be nice.